server/jetstream_cluster_test.go

```diff
@@ -9442,6 +9442,52 @@ func TestJetStreamClusterAccountInfoForSystemAccount(t *testing.T) {
 	}
 }
 
+func TestJetClusterStreamListFilter(t *testing.T) {
```

Comment:
Can be made into one test that tests single server and cluster, I have a few examples of that.

Reply:
Ok